to.etc.domui/src/main/resources/resources/themes/css-domui/670_wizard.frag.css

```diff
@@ -0,0 +1,34 @@
+/** Generic Wizard Component **/
+div.wizard-navbar {
```

Comment:
In DomUI the convention for class names is: make a short name for the component (like gwiz). Then use ui-gwiz-xxxx as class names. This makes clear where labels come from and prevents name clashes with other style sheets.